RxExample/RxExample/Examples/GitHubSignup/Views/GitHubSignupViewController.swift

```diff
@@ -108,9 +108,8 @@ class GitHubSignupViewController : ViewController {
                 let validationColor: UIColor
                 
                 if let valid = v.valid {
-                    validationColor = valid ? okColor : errorColor
-                }
-                else {
+                    validationColor = valid ? self.okColor : self.errorColor
+                } else {
```

Comment:
I think this would create a memory leak because of referencing strong `self`.

I'll merge this and try to play with this and see how to make this nicer but without leaks :)